inst/get_files_workflow.R

```diff
@@ -0,0 +1,34 @@
+devtools::load_all(".")
+
+test_gitstats <- create_gitstats() |>
+  set_github_host(
+    repos = c("r-world-devs/dbplyr", "r-world-devs/MegaStudy", "openpharma/visR")
+  )
+
+files_structure <- get_files_structure(
+  gitstats_obj = test_gitstats,
+  pattern = "\\.md"
```

Comment:
Let's assume we want to check specific repo structure, e.g. some Shiny dashboard. We often keep there logo or screenshots used in `About`/`Help` sections. `get_files_structure()` will nicely return a result. But when you try to `get_files_content()`, you will get error. 

It would be nice to handle it - if for some reason we can't get file content, `get_files_content()` should return NA so we can easily use it in final table with results. Below you can find some repro code:

```
files_structure <- get_files_structure(
  gitstats_obj = test_gitstats,
  pattern = "\\.png"
)

file_content <- purrr::map(files_structure[[1]], function(org) {
  purrr::map(org, function(repo_files) {
    get_files_content(
      gitstats_obj = test_gitstats,
      file_path = repo_files,
      verbose = FALSE
    )
  }) |>
    purrr::list_rbind()
}) |>
  purrr::list_rbind()
```

Reply:
That's a good point. I've fixed it for GitHub, but for GitLab it may take some more time, as it's workflow for getting file content differs substantially from GitHub. As the PR deals with adding new `get_files_structure()` I would address your concerns further in https://github.com/r-world-devs/GitStats/issues/467